src/main/java/io/github/syst3ms/skriptparser/expressions/CondExprStartsEnds.java

```diff
@@ -0,0 +1,63 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Main;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.lang.base.ConditionalExpression;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Check if a given string starts or ends with a certain string.
+ *
+ * @name Starts/Ends With
+ * @type CONDITION
+ * @pattern %strings% (start|end)[s] with %string%
+ * @pattern %strings% do[es](n't| not) (start|end) with %string%
+ * @since ALPHA
+ * @author Mwexim
+ */
+public class CondExprStartsEnds extends ConditionalExpression {
+
+    static {
+        Main.getMainRegistration()
+                .addExpression(CondExprStartsEnds.class,
+                        Boolean.class,
+                        true,
+                        "%strings% (1:start|2:end)[s] with %string%",
```

Comment:
In general, the default parse mark value should be 0, as convention.

Reply:
Then you'll have to change that in all commits I made until now.

Reply:
When there's a single parse mark, wouldn't that create issues? Since the parse mark is always 0 by default?